apps/test/unit/code-studio/components/SoundLibraryTest.js

```diff
@@ -0,0 +1,27 @@
+import { expect } from '../../../util/configuredChai';
+import React from 'react';
+import { shallow } from 'enzyme';
+import SoundLibrary from '@cdo/apps/code-studio/components/SoundLibrary';
+import sinon from 'sinon';
+
+describe('SoundListEntry', () => {
+  it('stops playing sound when user chooses a sound', () => {
+    const wrapper = shallow(
```

Comment:
I _thought_ that `componentWillMount` (or all lifecycle methods) don't get called with enzyme shallow-rendering, so in theory you're not creating a `sounds` object at all... but then, I'm not sure how you stub `stopAllAudio` below, if that's the case.  In any case, I wonder if you should use enzyme `mount` for this test.